metricbeat/module/system/_meta/config.full.yml

```diff
@@ -4,10 +4,10 @@
     - cpu
 
     # System Load stats
-    - load
+    #- load
 
     # Per CPU core stats
-    #- core
+    - core
```

Comment:
Also not sure about this one, I think we had it disabled by default in an effort to keep disk usage down?